src/docfx/publish/PublishModelBuilder.cs

```diff
@@ -18,8 +18,6 @@ internal class PublishModelBuilder
 
         public void MarkError(Document file)
         {
-            // TODO: If Error has a Document identifier, we can retrieve files with errors from
-            //       error log without explicitly call PublishModelBuilder.MarkError
             _filesWithErrors.Add(file);
```

Comment:
Leave this TODO: this is about decouple `PublishModelBuilder` from `Document` and use `FilePath` as the key to here

cc @928PJY

Reply:
> Change to hide fallback path as we discussed

I have hidden the fallback path